docker-exercises/exercise-4/frontend/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:20-alpine
+
+# Set the working directory
+WORKDIR /app
+
+# Copy package.json and install dependencies
+COPY package.json ./
+RUN npm install
+
```

Comment:
Simplify COPY commands, copy both package.json and package*.json. Use .dockerignore to stop node_modules being copied into the container.

Reply:
Done ✅